api-reference/10 UI Components/dxDrawer/1 Configuration/width.md

```diff
@@ -5,3 +5,17 @@
 Specifies the view's width.
 
 ---
+This property accepts a value of one of the following types:
+
+- **Number**  
+The width in pixels.
+
+- **String**  
+A CSS-accepted measurement of width. For example, `"55px"`, `"auto"`, `"inherit"`. Do not set the width in percent.
```

Comment:
'XX%' will work as expected: entire drawer element height will be 'XX%' of its parent' height.
imho we should not change this text.

https://codepen.io/DanIgnatov/pen/GRNLBPv?editors=1010